### ISSUE TITLE
About role="doc-cover"

### ISSUE THREAD

**Issue by coolharisha** [daisy/kb#22]
Whenever assign role="doc-cover" in the <img> tag, screen reader JAWS doesn't read the ALT text content and also doesn't intimate graphic. Please guide as it Screen reader issue or tagging issue

Code Example:

<img src="images/cover.jpg"
     role="doc-cover"
     alt="EPUB 3 Best Practices - O'Reilly - Pencil drawing of a goat">

**Comment by mattgarrish** [comment]
Apologies for the late response, but it could be either an AT issue or a combination of AT and reading system. If doc-cover isn't recognized, the implicit role of the img tag should take precedence.

A possible workaround might be to include an explicit fallback role, like this:

```role="doc-cover img"```

But I'm not sure JAWS handles multiple explicit roles any better, assuming it is JAWS that is the problem. You also aren't technically supposed to explicitly assign the implicit roles of elements, so it likely opens up other validation issues.

A similar issue was opened here: https://github.com/w3c/epubcheck/issues/1048

Closing this issue as I'll track any possible warnings we might need to make in the other.
